src/utils/errors.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable max-len */
+const invalidText =
+  'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
+const invalidToCase =
+  '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
+const invalidToCaseValue =
+  'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.';
+
```

Comment:
They should be named in `UPPER_CASE`, but even better it should be object with different texts as properties